GetVisibleServices/handler.ts

```diff
@@ -72,7 +72,8 @@ export function GetVisibleServicesHandler(
                 : arr
           );
           return ResponseSuccessJson({
-            items: servicesTuples
+            items: servicesTuples,
+            prev: servicesTuples[0]?.service_id ?? "-"
```

Comment:
may you add a comment on why this is needed?

Reply:
Done in https://github.com/pagopa/io-functions-app/pull/221/commits/bc1a6bce6873b7cbff7f3555ac8a0e1c539f5c64